src/Model/test.js

```diff
@@ -208,6 +209,88 @@ describe('CRUD', () => {
     expect(nextObserver).toHaveBeenCalledTimes(1)
     expect(completionObserver).toHaveBeenCalledTimes(1)
   })
+  it('can destroy a record and its children permanently', async () => {
+    const { database, projects, tasks, comments } = mockDatabase()
+
+    const project = await projects.create(mock => {
```

Comment:
i think it would be better to have the test create more than one record of each time to actually test that the entire record tree is deleted.

it would be appropriate to test this once - a unit test for fetchChildren helper, and then have quick&dirty tests for experimentalDelete* functions

Reply:
done